app/services/proofing/aamva/request/verification_request.rb

```diff
@@ -140,6 +140,20 @@ def user_provided_data_map
           }
         end
 
+        def state_id_number
```

Comment:
💡 idea, what if we added redacted logging of this number (like this) https://github.com/18F/identity-idp/blob/622f9c52e0456f6e1f178f7aa92c30c51718d5d6/app/forms/new_phone_form.rb#L130-L132

that way we could spot-check the number-letter format of what we're seeing vs various state formats we can online? http://www.diogenesllc.com/stdlformats.html

Reply:
Great idea - done in 44a16ff